lib/utils/twacode.dart

```diff
@@ -752,7 +754,7 @@ class TwacodeRenderer {
         child: Column(
           children: fileIds
               .map((id) => (id.isNotEmpty)
-              ? FileTile(fileId: id)
+              ? FileTile(fileId: id, isMyMessage: isMyMessage ?? false)
```

Comment:
isMyMessage: parentStyle.color == Colors.black ? false : true,

Reply:
You're right! Agreed with that

Reply:
You're right. We can handle it by this solution, even though it's similar to `hard code` (we are fixed the color checking by `Colors.black`). Anyway, I prefer dynamic parameter